perf/perf_uprobe.py

```diff
@@ -24,7 +24,7 @@
 from avocado.utils.software_manager import SoftwareManager
 
 
-class perfUprobe(Test):
+class PerfUprobe(Test):
```

Comment:
why this change

Reply:
That is python class convention i followed here Praveen

Reply:
yes i know but it should be documented in commit message :)  ideal way it should go as seperate commit

Reply:
Oh that i donno so i sent it in the same commit

Reply:
should be fine but please document as part of commit message :)

Reply:
sure Praveen